lib/matplotlib/projections/polar.py

```diff
@@ -1100,21 +1108,59 @@ def get_theta_direction(self):
         return self._direction.get_matrix()[0, 0]
 
     def set_rmax(self, rmax):
+        """
+        Set the outer radial limit.
+
+        Parameters
+        ----------
+        rmax : scalar
```

Comment:
```suggestion
        rmax : float
```

https://matplotlib.org/devdocs/devel/documenting_mpl.html?highlight=writing%20documentation#parameter-type-descriptions

> Use float for a type that can be any number.